NSS-Therapeutic-Raccoons/WorkforceManagement
# Df departments view all

## Link to Ticket
Closes # [Link](https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/6)

## Description of Proposed Changes
- Modified Departments Index() to display a count of employees
- Created integration test for Index

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <df-departments-viewAll>
run the tests and verify in browser
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Departments/Index.cshtml
* DepartmentIndexViewModel.cs
* DepartmentTests.cs

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot
